src/macros.rs

```diff
@@ -0,0 +1,29 @@
+#[macro_export]
```

Comment:
Once this PR gets merged, we'll probably need to update our readme. Would you wanna do that @JustBobinAround?

Reply:
Yep, will work on it.